QUEST_LV_0100.tsv

```diff
@@ -5631,9 +5631,9 @@ QUEST_LV_0100_20150317_005630	Return to Houberg and release his last soul and wi
 QUEST_LV_0100_20150317_005631	Defeat the Stone Whale
 QUEST_LV_0100_20150317_005632	Find Goddess in Mage Tower 5th Floor
 QUEST_LV_0100_20150317_005633	Look for the Goddess in Mage Tower 5th Floor
-QUEST_LV_0100_20150317_005634	Destroy the spell suppressor that Helgasercle installed
-QUEST_LV_0100_20150317_005635	Destroy all the spell suppressors that Helgasercle installed around Mage Tower 5th Floor
-QUEST_LV_0100_20150317_005636	Defeat Helgasercle that invaded the Mage Tower
+QUEST_LV_0100_20150317_005634	$Destroy the Magic Suppressor prepared by Helgasercle
+QUEST_LV_0100_20150317_005635	$Destroy all the magic suppressors that Helgasercle prepared on the 5th floor of the Mage Tower.
+QUEST_LV_0100_20150317_005636	$Find and defeat Helgasercle who invaded in the Mage Tower.
```

Comment:
She's the boss at the end of the end of the quest line, of which you fight her in a large "final" area infront of a large locked (?) door.  90% sure she's just invading the Mage Tower as a whole, as you have to interact with lots of obstacles she puts up along the way.

(I see that you changed this, but I'm just clarifying)